pod/perldelta.pod

```diff
@@ -417,7 +417,7 @@ XXX Generate this with:
 
 If you find what you think is a bug, you might check the perl bug database
 at L<https://github.com/Perl/perl5/issues>.  There may also be information at
-L<http://www.perl.org/>, the Perl Home Page.
+L<https://www.perl.org/>, the Perl Home Page.
```

Comment:
unfortunately all previous delta's went with http://, so fix this in the general template to affect all future perldeltas